src/components/Exchange/SwapBox.js

```diff
@@ -1058,36 +1073,40 @@ export default function SwapBox(props) {
   };
 
   const renderErrorModal = () => {
-    const inputCurrency = fromToken.address === AddressZero ? "ETH" : fromToken.address;
+    const nativeToken = getNativeToken(chainId).symbol;
+    const inputCurrency = fromToken.address === AddressZero ? nativeToken : fromToken.address;
     let outputCurrency;
     if (isLong) {
-      outputCurrency = toToken.address === AddressZero ? "ETH" : toToken.address;
+      outputCurrency = toToken.address === AddressZero ? nativeToken : toToken.address;
```

Comment:
small request, can we use `nativeToken.symbol` instead of `nativeToken`? all (i guess) token variables here represent token object, so other developers might not expect it to be string. just for improved readability, what do you think?

Reply:
or name it nativeTokenSymbol?